Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -18913,7 +18913,9 @@
 		};
 		5C400E6A29DB8AB500446F6F /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			platformFilter = ios;
+			platformFilters = (
+				ios,
+			);
```

Comment:
Ditto.

Reply:
I actually think it's OK to land these changes. Otherwise, Xcode will keep making them whenever you modify this project.

Reply:
Does older Xcode versions support this, or do we not need to maintain compatibility?

Reply:
Yes they support this.